node/modules/dtypes/beacon.go

```diff
@@ -0,0 +1,6 @@
+package dtypes
+
+type DrandConfig struct {
```

Comment:
I tried defining this in `drand.go` and aliasing, but hit an import cycle

Reply:
No, this is the right place for it.

Reply:
Can't you use just this type in `chain/beacon/drand`? Would that also lead to import cycle?

Reply:
ah, that works, thanks! I thought I tried it that way but I guess not :)